New reference image

pp-mo opened the pull request:
Fixes iris.test.test_plot.TestSymbols.test_cloud_cover,
failing since  matplotlib 3.2.0.

See : https://github.com/SciTools/iris/pull/3682

pp-mo committed:
New reference image for iris.test.test_plot.TestSymbols.test_cloud_cover with matplotlib 3.2.0.

trexfeathers reviewed (commented):
Thanks @pp-mo! Please can you explain the Travis failures for this? Seems like there is some confusion over file extensions.

pp-mo committed:
Updated content listing file to appease run_test.TestListing

pp-mo commented:
Fixed the listing file, and rebased onto #30 (effectively)
This should check out, but when #30 is merged, it may need a further re-rebase.  Not sure of how that will work out ...

trexfeathers reviewed (approved):
All good, thanks @pp-mo
